noir-projects/noir-protocol-circuits/crates/private-kernel-lib/src/components/private_kernel_circuit_output_validator.nr

```diff
@@ -390,20 +368,17 @@ impl PrivateKernelCircuitOutputValidator {
             offsets.unencrypted_logs_hashes,
             storage_contract_address
         );
-        validate_array_appended_reversed_scoped(
+        validate_array_appended(
```

Comment:
This adds another 3k to inner :(
Same as above, will investigate and try to reduce it in the next pr.